weave/table.py

```diff
@@ -28,3 +31,23 @@ def __getitem__(self, key: int) -> dict:
 
     def __iter__(self) -> Iterator:
         return iter(self.rows)
+
+    def __eq__(self, other: Any) -> bool:
+        return self.rows == other
+
+    def add(self, row: dict) -> None:
```

Comment:
i would use the same names as python (append, pop) - i suspect that Table might end up wanting to implement the entire list protocol

Reply:
Well this is the correct question. Specifically, a Table is assumed to be a List[Dict], so that is one distinction. But other than that, we probably always want to use Table. Again, table is purely an implementation detail, and should not be something the user has to think about.